client/src/components/Atoms/ListAndDetail.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { ReactNode } from 'react'
+import styled from 'styled-components'
+import { Button, Card, Classes, Colors, InputGroup } from '@blueprintjs/core'
+
+/**
+ * A set of components to render a list-and-detail panel
+ *
+ * Example usage:
+ *
+ *  <ListAndDetail>
+ *    <List>
+ *      <ListItem>Item 1</ListItem>
+ *      <ListItem selected>Item 2</ListItem>
+ *    </List>
+ *    <Detail>
+ *      {detailsForSelectedListItem}
+ *    </Detail>
+ *  </ListAndDetail>
+ */
+
+export const ListAndDetail = styled(Card)`
+  display: grid;
+  grid-template-columns: 1fr 3fr;
```

Comment:
I think to maintain visual consistency, the change in content width of the detail view shouldn't change the layout of the list/detail panels. I'm specifically thinking of the case of adding many tally sheet tabs:

https://user-images.githubusercontent.com/530106/199078316-70544926-613b-4701-9d47-ab25222839ab.mov